File: basic/src/main/java/com/ding/basic/bean/SearchResult.java
package com.ding.basic.bean;

import java.util.List;

/**
 * @author lijun Lee
 * @desc 搜索 API Result
 * @mail jun_li@dingyuegroup.cn
 * @data 2017/12/5 11:54
 */

public class SearchResult {

    private List<HotWordBean> hotWords;

    private List<SearchOperations> operations;

    public List<HotWordBean> getHotWords() {
        return hotWords;
    }

    public void setHotWords(List<HotWordBean> hotWords) {
        this.hotWords = hotWords;
    }

    public List<SearchOperations> getOperations() {
        return operations;
    }

    public void setOperations(List<SearchOperations> operations) {
        this.operations = operations;
    }

    @Override
    public String toString() {
        return "SearchResult{" +
                "hotWords=" + hotWords +
                ", operations=" + operations +
                '}';
    }
}

File: basic/src/main/java/com/ding/basic/bean/SearchResult.kt
package com.ding.basic.bean

import java.io.Serializable

/**
 * Desc 热词和推荐实体类
 * Author JoannChen
 * Mail yongzuo_chen@dingyuegroup.cn
 * Date 2018/9/19 0019
 */
data class SearchResult(
        var hotWords: ArrayList<HotWordBean>?,
        var operations: ArrayList<SearchOperations>?) : Serializable